git.mk

```diff
@@ -367,6 +367,7 @@ $(srcdir)/.gitignore: Makefile.am $(top_srcdir)/git.mk $(top_srcdir)/configure.a
 			"*~" \
 			".*.sw[nop]" \
 			".dirstamp" \
+			".vscode/" \
```

Comment:
For what is that visual studio code entry good?

Reply:
@fogbank please remove this entry. You can configure a global ~/.gitconfig to deal with any special configuration in your machine.